exporter/opentelemetry-exporter-datadog/tests/test_datadog_exporter.py

```diff
@@ -667,3 +673,50 @@ def test_service_name_fallback(self):
 
         span = datadog_spans[0]
         self.assertEqual(span["service"], "fallback_service_name")
+
+    @unittest.skipUnless(
+        hasattr(os, "fork") and sys.version_info >= (3, 7),
+        "needs *nix and minor version 7 or later",
+    )
+    def test_exports_with_fork(self):
```

Comment:
This test is effectively the same sort of one from https://github.com/open-telemetry/opentelemetry-python/pull/2242 with as few changes as possible.